mods/weather/init.lua

```diff
@@ -1,11 +1,20 @@
 -- Disable by mapgen or setting
 
 local mg_name = minetest.get_mapgen_setting("mg_name")
-if mg_name == "v6" or mg_name == "singlenode" or
-		minetest.settings:get_bool("enable_weather") == false then
+if minetest.settings:get_bool("enable_weather") == false then
 	return
 end
 
+if mg_name == "v6" or mg_name == "singlenode" then
+	-- set a default shadow intensity for mgv6 and singlenode
+	minetest.register_on_joinplayer(function(player)
+		local light = player:get_lighting()
+		light.shadows.intensity = 0.33
+		player:set_lighting(light)
```

Comment:
it is simpler to just write player:set_lighting({ shadows = { intensity = 0.33 } }). The API call keeps values you don't pass.

Reply:
Didn't check that... Yeah, that's much cleaner.